### ISSUE TITLE
Possibility to set the language

### ISSUE THREAD

**Issue by fabiankuehne** [infusion-code/angular-maps#108]
Is it possible to add the language to the loaderConfig or better, change it at runtime?

<script type='text/javascript' src='http://www.bing.com/api/maps/mapcontrol?callback=GetMap**&setMkt=fr-FR&setLang=fr**' async defer></script>

I think the api has this feature like in the link above, but i can't find it in angular-maps.

**Comment by thor-schueler** [comment]
Yes, should be easily possible. do you want to try to create a pull request? I'd suggest adding a language option field to the MapOptions.....
